website/content/en/wgs/app-development/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: App Development Working Group
```

Comment:
The content in these two new pages is copied/pasted from what we have in the root folder `/app-development-wg`. I couldn't find a way to re-use them directly, but I saw the other WGs do the same, so maybe we don't have a way to solve that.

Reply:
I think that is ok for now